src/patternfly/layouts/Page/styles.scss

```diff
@@ -130,29 +119,31 @@ $nav: "pf-c-nav";
     display: flex;
     align-items: flex-end;
   }
-
-  .pf-c-avatar {
-    margin-left: var(--pf-l-page__header-avatar--MarginLeft);
-  }
 }
 
 // Brand
 .pf-l-page__header-brand {
-  align-items: flex-end;
   padding-top: var(--pf-l-page__header-brand--PaddingTop);
   padding-bottom: var(--pf-l-page__header-brand--PaddingBottom);
   padding-left: var(--pf-l-page__header-brand--PaddingLeft);
 
   @media (min-width: $pf-global--breakpoint--md) {
     flex: 0 0 var(--pf-l-page__header-brand--FlexBasis--lg);
-    align-items: flex-end;
     align-self: stretch;
     padding-right: var(--pf-l-page__header-brand--PaddingRight); // set padding right here to allow mobile view to accomodate tools
     background-color: var(--pf-l-page__header-brand--BackgroundColor--md);
   }
 
-  .pf-m-patternfly-logo {
-    height: var(--pf-l-page__header-brand-logo--Height);
+  // Sidebar Toggle
+  .pf-c-button {
```

Comment:
Also a comment here to explain why we are changing the pf-c-button within the sidebar.
Just a thoight: we don't want anyone to add a normal btn there, but is this a dangerous move? what is a user wants to add a btn to this section and make it look like a button.

Reply:
Added a container in the layout for the toggle.